tests/Microsoft.Bot.Builder.Dialogs.Declarative.Tests/SchemaMergeTests.cs

```diff
@@ -25,19 +26,9 @@ namespace Microsoft.Bot.Builder.Dialogs.Declarative.Tests
     /// npm i -g @microsoft/botframework-cli
     /// bf plugins:install @microsoft/bf-dialog
     /// </remarks>
-    [TestClass]
     public class SchemaMergeTests
     {
-        public static ResourceExplorer ResourceExplorer { get; set; }
-
-        public static JSchema Schema { get; set; }
-
-        public static IEnumerable<object[]> Dialogs { get; set; }
-
-        public TestContext TestContext { get; set; }
-
-        [ClassInitialize]
```

Comment:
I think we should use IClassFixture replacing ClassInitialize so it is executed once before running all the tests in this class.

Reply:
I couldn't implement _IClassFixture_ in this class as it needs a static constructor.